### PR TITLE
Add product scan to legal-auto

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by kraih** [comment]
I'll be testing this in production on the legaldb over the next few days.

File: legal-auto.py [hunk 1]
```diff
@@ -178,6 +208,108 @@ def _set_review(self, req, state):
                                      by_user=self.review_user, message=self.message)
         self.delete_from_db(req.reqid)
 
+    def _pkl_path(self):
+        CACHE_DIR = os.path.expanduser('~/.cache/osc-plugin-factory')
```
Comment (jberry-suse) [inline legal-auto.py]: nit: use the new project name and perhaps xdg as done elsewhere.

Comment (coolo) [reply legal-auto.py]: yeah - these are changes done in june 2017 (and used in production since then ;)

**Comment by kraih** [comment]
Seems to be working smoothly so far. Running it once a day around midnight with a systemd timer.

File: legal-auto.py (hunk 1 above)
Comment (lnussel) [reply legal-auto.py]: could you please fix this nevertheless?

Comment (jberry-suse) [reply legal-auto.py]: why merge?

**Pull request merged**